components/json-pointer.js

```diff
@@ -0,0 +1,50 @@
+// json-pointer.js
+
+var _ = require('underscore');
+var jsonpointer = require('jsonpointer');
+
+var promiseOutput = require('../src/promise-output');
+var componentFactory = require('../src/noflo-component-factory');
+
+/**
+ * Takes the value of the get pointer for some input and sets that value on the
+ * input at the set pointer. If there is not set pointer provided, this will
+ * return the get pointer value, otherwise it will return the input object.
```

Comment:
Maybe we should have a subdirectory for some of these helper components like parseJson or json-pointer since they are wrapped like pipeline components but aren't really providing pipeline functionality.